plugins/provenance/server/resource.py

```diff
@@ -329,7 +329,9 @@ def fileSaveHandler(self, event):
                                        level=AccessType.READ, force=True)
         if not item:
             return
-        prevFile = self.model('file').load(curFile['_id'])
+        # should all of these sort of events be forced, since a save has to be
+        # granted first?
+        prevFile = self.model('file').load(curFile['_id'], force=True)
```

Comment:
Hm, you're right. Although, it's counterintuitive and inconsistent -- I think a good fix for that would be to change `parentsToRoot` such that it doesn't call `filter` when `force=True`.